### PR TITLE
#CU-81tm1p Create BoxBase component

### PR DESCRIPTION
**Pull request by binhtt13122000**
###### Align-items
image
###### Justify-content:
image
###### Flex-direction:
image
###### Color:
image
###### Size:
image

### CODE REVIEW

**Comment by binh1298** [comment]
Task linked: [CU-81tm1p Box Component](https://app.clickup.com/t/81tm1p)

File: src/components/Boxs/BoxBase/index.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    pr?: number;
+    pb?: number;
+    pl?: number;
+    width?: number;
+    height?: number;
+    minWidth?: number;
+    maxWidth?: number;
+    minHeight?: number;
+    maxHeight?: number;
+}
```
Comment (binh1298) [inline src/components/Boxs/BoxBase/index.tsx:43]: I like your effort but have you checked the material interfaces for Box? 
Please explore the typings of Box by **CTRL click** on it and use the pre-written types appropriately
node_modules\@material-ui\system\index.d.ts
image

**Review by binh1298 — COMMENTED** [review]
Exceptional effort but it's even better to work smart ;)

**Review by binh1298 — APPROVED** [review]

**Comment by binhtt13122000** [comment]
image
I will change children item and delete background color

**Comment by binhtt13122000** [comment]
image
I added props of typography.

**Review by binh1298 — APPROVED** [review]
Very good!

**Pull request merged**